### PR TITLE
add documentation for Toolbox tests.

### PR DESCRIPTION
**Pull request by mkudlej**

### CODE REVIEW

File: doc/TOOLBOX.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  toolbox:
+    cmd: "podman"
+    podman_image: "" # hash of Toolbox image from `podman image list`
+    destination_endpoint: "" # url to 3scale API endpoint of second tenant
+    destination_provider_key: "" # access token for `destination_endpoint`
+    machine_ip: "" # RHEL machine IP
+    ssh_user: "" # SSH user for access to RHEL machine who is able to run podman images
+    ssh_passwd: "" # SSH password of `ssh_user`
+
+  threescale:
```
Comment (pehala) [inline doc/TOOLBOX.md:28]: Is this needed even if you are logged in to openshift?

File: doc/TOOLBOX.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    podman_cert_name: "ca-bundle.crt" # Certificate bundle to verify TLS connections. This can be disabled by configuration option 'ssl_verify: false'.
+```
+
+## Run Toolbox tests
+
+There is an extra parameter required for Toolbox tests - `--toolbox`
+Example:
+
+```sh
+pipenv run pytest --toolbox testsuite/tests/toolbox/
```
Comment (pehala) [inline doc/TOOLBOX.md:49]: I think it would be better to use `make` in our documentation

File: doc/TOOLBOX.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Before You Run It
+
+### Requirements
+
+* RHEL 7/8 machine
+* docker/podman installed on RHEL machine
+* sshd installed and configured on RHEL machine
+* Toolbox image pulled on RHEL machine
+* two 3scale tenants adn related access tokens
```
Comment (pehala) [inline doc/TOOLBOX.md:11]: typo (adn)

File: doc/TOOLBOX.md (hunk 2 above)
Comment (mkudlej) [reply doc/TOOLBOX.md:49]: @pehala I don't use make for running tests. Do you have any hint for running Toolbox tests?

File: doc/TOOLBOX.md (hunk 1 above)
Comment (mkudlej) [reply doc/TOOLBOX.md:28]: It is not needed. I would like to just stress it out. So it is clear that Toolbox uses also these options. I am going to add there some text to clear it more.

File: doc/TOOLBOX.md (hunk 2 above)
Comment (mdujava) [reply doc/TOOLBOX.md:50]: https://github.com/3scale-qe/3scale-tests/blob/c2c8d3237c47656d649be096ac412993e439cf1e/Makefile#L59-L60
simple `make toolbox`

Comment (mkudlej) [reply doc/TOOLBOX.md]: Done

**Review by mganisin — APPROVED** [review]

File: doc/TOOLBOX.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Before You Run It
+
+### Requirements
+
+* RHEL 7/8 machine
+* docker/podman installed on RHEL machine
+* sshd installed and configured on RHEL machine
+* Toolbox image pulled on RHEL machine
+* two 3scale tenants and related access tokens
```
Comment (mdujava) [inline doc/TOOLBOX.md:11]: This is optional, if not supplied and logged in to openshift, brand new tenants are created.

Comment (mkudlej) [reply doc/TOOLBOX.md:11]: I don't see this sentence as contradiction. Toolbox tests require 2 tenants with related access tokens. It doesn't matter if they are created by magic in testsuite, they are from different OCPs or even from SaaS. As it is mentioned below, `threescale` section is optional.

Comment (mdujava) [reply doc/TOOLBOX.md:11]: It's in section `Before you run it (tests)`

same as destination_endpoint and destination_provider_key are optional.

**Review by pehala — APPROVED** [review]

**Pull request merged**